rewrite-java-21/build.gradle.kts

```diff
@@ -24,6 +24,7 @@ dependencies {
     implementation("io.micrometer:micrometer-core:1.9.+")
     implementation("io.github.classgraph:classgraph:latest.release")
     implementation("org.ow2.asm:asm:latest.release")
+    implementation("org.openrewrite:rewrite-templating:latest.release")
```

Comment:
Rather than add a dependency here I think it's likely best to add copies of the field and method used here in the rewrite-java-21 parser itself. This dependency contains an annotation processor that would be somewhat odd to pull in here.

Reply:
Good point. I'll change the approach.